.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,68 @@
+name: Bug Report
+description: Report a bug in Karpenter
+labels: bug
+body:
+  - type: textarea
+    id: version
+    attributes:
+      label: Version
+      description: |
+        https://github.com/aws/karpenter/releases
```

Comment:
Since we don't use karpenter-core releases yet, I think this is fine. We should change this when we start adding those.